lang/semgrep-grammars/src/semgrep-ruby/grammar.js

```diff
@@ -69,6 +69,11 @@ module.exports = grammar(standard_grammar, {
     $.semgrep_ellipsis_followed_by_newline
   ]),
 
+  conflicts: ($, previous) => previous.concat([
+    [$.keyword_pattern, $.pair]
```

Comment:
would be nice each time we add a conflict to have a short comment before giving an example of such conflict.